### PR TITLE
Location sharing frontend

### LINKED ISSUE #1478: Share a location
**Is your feature request related to a problem? Please describe.**
Sometimes, I would like to let my family know my current whereabouts without typing it as I might not even know the correct address.

**Describe the solution you'd like**
Give me the ability to send a message with messageParameter of type "location" with lat and lng parameters.

### PR DESCRIPTION
**Pull request by ma12-co**
Fix #1478 

Screenshot from 2021-05-07 12-08-39

### CODE REVIEW

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Location.vue [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<a :href="mapLink"
+		target="blank"
```
Comment (nickvergessen) [inline src/components/MessagesList/MessagesGroup/Message/MessagePart/Location.vue:24]:
```suggestion
		target="_blank"
		rel="noopener noreferrer"
```

**Comment by ma12-co** [comment]
-[ ] Missing location name

File: src/mainAdminSettings.js [hunk 2]
```diff
@@ -25,6 +25,12 @@ import AdminSettings from './views/AdminSettings'
 
 // Styles
 import '@nextcloud/dialogs/styles/toast.scss'
+import 'leaflet/dist/leaflet.css'
```
Comment (nickvergessen) [inline src/mainAdminSettings.js:28]: shouldnt be needed here :D

**Comment by nickvergessen** [comment]
Can you update the screenshot?

**Comment by ma12-co** [comment]
New design with map spanning the whole with of the message.

Screenshot from 2021-05-10 16-22-41

**Pull request merged**